deploy/configs/applications/modules.yaml

```diff
@@ -45,6 +45,7 @@ modules:
       - py-bluepymm
       - py-bluepyopt
       - py-efel
+      - py-entity-management
```

Comment:
Here we are just whitelisting modules. In order to actually trigger build, we have to add entry into [spack/deploy/packages/bbp-packages.yaml](https://github.com/BlueBrain/spack/blob/develop/deploy/packages/bbp-packages.yaml).

Once CI finishes, you can test the PR modules on BB5 as:

```
kumbhar@bbpv2:~$ source  /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/373/config/modules.sh
kumbhar@bbpv2:~$ module av

------------------------------------------ /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/373/modules/compilers/2019-05/tcl/linux-rhel7-x86_64 ------------------------------------------
gcc/6.4.0                            gcc/8.2.0                            intel-parallel-studio/cluster.2018.3 pgi/19.4
gcc/7.3.0                            intel/18.0.3                         llvm/7.0.0

-------------------------------------------- /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/373/modules/tools/2019-05/tcl/linux-rhel7-x86_64 --------------------------------------------
allinea-forge/19.0.5 doxygen/1.8.15       ffmpeg/3.2.4         intel-mpi/2018.1.163 mvapich2/2.3
cmake/3.13.0         emacs/26.1           hpe-mpi/2.16         ispc/1.10.0

------------------------------------- /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/373/modules/external-libraries/2019-05/tcl/linux-rhel7-x86_64 --------------------------------------
cuda/9.0.176  cuda/9.2.88   cudnn/7.3     python/2.7.15 python/3.6.5  spark/2.4.0

-------------------------------------- /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/373/modules/serial-libraries/2019-05/tcl/linux-rhel7-x86_64 ---------------------------------------
boost/1.68.0/serial         hdf5/1.10.4/serial          highfive/2.0/serial         neuron/7.6.6/python3/serial python-dev/0.1/python3
gmsh/3.0.6/serial           highfive/1.6/serial         neuron/7.6.6/python2/serial python-dev/0.1/python2      tensorflow/1.12.0/python3

------------------------------------- /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/373/modules/parallel-libraries/2019-05/tcl/linux-rhel7-x86_64 --------------------------------------
hdf5/1.10.4/parallel          neuron/7.6.6/python3/parallel petsc/3.10.2/parallel         trilinos/xsdk-0.4.0-rc1
highfive/2.0/parallel         omega-h/9.26.3/parallel       py-mpi4py/3.0.0/python3

---------------------------------------- /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/373/modules/applications/2019-05/tcl/linux-rhel7-x86_64 -----------------------------------------
brainbuilder/0.10.5/python3            neurodamus-neocortex/0.1/python3       py-bglibpy/4.0.17/python3              reportinglib/2.5.1
brayns/0.8.0/python3                   neurodamus-thalamus/0.1/knl/python3    py-bluepymm/0.6.38/python2             reportinglib/develop
connectome-tools/0.3.2/python3         neurodamus-thalamus/0.1/python3        py-bluepymm/0.6.38/python3             spykfunc/0.13.2/python3
functionalizer/3.12.1                  parquet-converters/0.3                 py-bluepyopt/1.6.56/python2            steps/3.4.1/python3/parallel
neurodamus-hippocampus/0.1/python3     placement-algorithm/2.0.8/python3      py-bluepyopt/1.6.56/python3            synapsetool/0.4.1/parallel
neurodamus-hippocampus/0.2/knl/python3 psp-validation/0.1.12/python3          py-efel/3.0.22/python2                 touchdetector/4.4.2
neurodamus-hippocampus/0.2/python3     py-basalt/0.2.4/python3                py-efel/3.0.22/python3
neurodamus-neocortex/0.1/knl/python3   py-bglibpy/4.0.17/python2              py-pytouchreader/1.4.2/python3
```

There is no `py-entity-management` module because it's not build yet:

```
kumbhar@bbpv2:~$ ll /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/373/deploy/applications/latest/linux-rhel7-x86_64/gcc-6.4.0/
total 0
drwxrwxr-x+ 18 bbprelman bbp 4096 May 23 15:46 ./
drwxrwxr-x+  3 bbprelman bbp 4096 May 23 15:41 ../
drwxr-sr-x+  5 bbprelman bbp 4096 May 23 15:42 brainbuilder-0.10.5-umc62ffidx/
drwxr-sr-x+  5 bbprelman bbp 4096 May 23 15:42 connectome-tools-0.3.2-yamiti6xxs/
drwxr-sr-x+  5 bbprelman bbp 4096 May 23 15:46 placement-algorithm-2.0.8-exdipiyurr/
drwxr-sr-x+  5 bbprelman bbp 4096 May 23 15:45 psp-validation-0.1.12-2wytl2tuzc/
drwxr-sr-x+  4 bbprelman bbp 4096 May 23 15:45 py-bglibpy-4.0.17-ph3dvyaubc/
drwxr-sr-x+  4 bbprelman bbp 4096 May 23 15:44 py-bglibpy-4.0.17-rkdxnlvb5b/
drwxr-sr-x+  4 bbprelman bbp 4096 May 23 15:41 py-bluepy-0.13.3-6kwmu36iwe/
drwxr-sr-x+  4 bbprelman bbp 4096 May 23 15:45 py-bluepy-0.13.3-t47dvwim6l/
drwxr-sr-x+  5 bbprelman bbp 4096 May 23 15:45 py-bluepymm-0.6.38-q72pm6kg2f/
drwxr-sr-x+  5 bbprelman bbp 4096 May 23 15:46 py-bluepymm-0.6.38-qophno27n2/
drwxr-sr-x+  5 bbprelman bbp 4096 May 23 15:45 py-bluepyopt-1.6.56-3mb7ivmxlv/
drwxr-sr-x+  5 bbprelman bbp 4096 May 23 15:46 py-bluepyopt-1.6.56-yckytq62xe/
drwxr-sr-x+  5 bbprelman bbp 4096 May 23 15:46 py-region-grower-0.1.2-i4ura3kwvp/
drwxr-sr-x+  4 bbprelman bbp 4096 May 23 15:46 py-tns-1.0.8-3nh2d3ebj5/
drwxr-sr-x+  4 bbprelman bbp 4096 May 23 15:42 py-voxcell-2.5.6-phdgziap6a/
drwxr-sr-x+  6 bbprelman bbp 4096 May 23 15:44 spykfunc-0.13.2-vels276vum/
```